indico/htdocs/js/indico/jquery/ajaxqbubble.js

```diff
@@ -0,0 +1,80 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    $.widget('indico.ajaxqbubble', {
+        options: {
+            url: null,
+            type: 'GET',
+            cache: false,
+            success: null
+        },
+
+        _create: function() {
+            var self = this;
+            var defaultQtipOptions = {position: {effect: false}, content: {text: $T.gettext('Loading..')}};
```

Comment:
Why are we hard-setting `defaultQtipOptions`? We should be able to specify how you want the `qbubble` to look even if you load it via AJAX.